src/SFA.DAS.Admin.Roatp.Web/Controllers/ApprenticeshipUnitsUpdateController.cs

```diff
@@ -20,10 +20,10 @@ public async Task<IActionResult> Index(int ukprn, CancellationToken cancellation
         if (organisationResponse == null) return RedirectToRoute(RouteNames.Home);
 
         var containsApprenticeshipUnits =
-            organisationResponse.AllowedCourseTypes.Any(a => a.CourseTypeId == CourseTypes.ApprenticeshipUnitId);
+            organisationResponse.Content!.AllowedCourseTypes.Any(a => a.CourseTypeId == CourseTypes.ApprenticeshipUnitId);
```

Comment:
create a variable for organisationResponse.Content! and use that in the controller instead of all the references